### PR TITLE
Add executors board

### PR DESCRIPTION
**Pull request by malomarrec**
## Test plan

Just a workflow

### CODE REVIEW

File: .github/workflows/label-move.yml [hunk 1]
```diff
@@ -274,3 +274,31 @@ jobs:
...
             }
           }' -f project=$PROJECT_ID -f node_id=$NODE_ID
+  executors-board:
+    runs-on: ubuntu-latest
+    env:
+      PROJECT_ID: PN_kwDOADy5QM4ADYmX # https://github.com/orgs/sourcegraph/projects/267
+      GITHUB_TOKEN: ${{ secrets.GH_PROJECTS_ACTION_TOKEN }}
+    steps:
+    - name: Get issue if relevant
+      if: ${{ contains(github.event.issue.labels.*.name, 'user-code-execution') }}
```
Comment (eseliger) [inline .github/workflows/label-move.yml:284]: should we also add rfc-696 or should we just make sure to label those with user-code-execution, too?

**Review by eseliger — APPROVED** [review]

File: .github/workflows/label-move.yml (hunk 1 above)
Comment (malomarrec) [reply .github/workflows/label-move.yml:284]: I think we should label all executors ticket (from now on) with `user-code-execution`, but I'm fine backing some more robustness in the system as well :)

Comment (eseliger) [reply .github/workflows/label-move.yml:284]: no that SGTM :)

Comment (malomarrec) [reply .github/workflows/label-move.yml:284]: I think we should label all executors ticket (from now on) with `user-code-execution`, but I'm fine backing some more robustness in the system as well :)

Comment (malomarrec) [reply .github/workflows/label-move.yml]: Added

**Pull request merged**